src/test/java/com/example/demo/controller/PostControllerTest.java

```diff
@@ -0,0 +1,200 @@
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.*;
+import static org.springframework.restdocs.payload.JsonFieldType.*;
+import static org.springframework.restdocs.payload.PayloadDocumentation.*;
+import static org.springframework.restdocs.request.RequestDocumentation.*;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.restdocs.RestDocumentationExtension;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import com.epages.restdocs.apispec.MockMvcRestDocumentationWrapper;
+import com.example.demo.dto.post.PostCreateRequestDto;
+import com.example.demo.dto.post.PostFindResponseDto;
+import com.example.demo.dto.post.PostsFindResponseDto;
+import com.example.demo.model.member.Member;
+import com.example.demo.model.member.Social;
+import com.example.demo.model.post.Genre;
+import com.example.demo.model.post.Post;
+import com.example.demo.service.PostService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@WebMvcTest(PostController.class)
+@WithMockUser
+@ExtendWith({MockitoExtension.class, RestDocumentationExtension.class})
+@AutoConfigureRestDocs
+class PostControllerTest {
+
+	@Autowired
+	@SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
+	private MockMvc mockMvc;
+
+	@Autowired
+	@SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
+	private ObjectMapper mapper = new ObjectMapper();
+
+	@MockBean
+	private PostService postService;
+
+	private final String musicId = "musicId";
+	private final String musicName = "musicName";
+	private final String musicUrl = "musicUrl";
+	private final String albumCoverUrl = "albumCoverUrl";
+	private final Genre genre = Genre.DANCE;
+	private final String singer = "hype";
+	private final boolean isPossibleBattle = true;
+	private final String content = "comment";
+	private final Member member = createMember();
+
+	@Test
+	void 성공_음악_공유_게시글을_등록할_수_있다() throws Exception {
+		// given
+		PostCreateRequestDto postCreateRequestDto = PostCreateRequestDto.builder()
```

Comment:
이것도 멤버 만드는 것 처럼 빼는 것도 좋을 것 같아요 테스트 코드에선 가독성이 제일 중요하다고 생각됩니다!
```suggestion
		PostCreateRequestDto postCreateRequestDto = getPostCreateRequest();
```
이런 식이면 요청을 받았다는 명확한 표현도 되는 것 같습니다.